modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/volumetricdatalayer/v0.1.0.json

```diff
@@ -0,0 +1,147 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/atlas/volumetricdatalayer/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/transformableobject/v1.0.0",
+    "{{base}}/schemas/neurosciencegraph/commons/brainlocation/v0.1.1",
+    "{{base}}/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.4"
```

Comment:
Add the {{base}}/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.4 schema as part of this PR.